src/preact/parse-props.js

```diff
@@ -170,9 +168,9 @@ function parsePropDefs(Ctor, props, propDefs, element, mediaQueryProps) {
     // as separate properties. Thus in a carousel the plain "children" are
     // slides, and the "arrowNext" children are passed via a "arrowNext"
     // property.
-    const nodes = realChildNodes(element);
+    const nodes = realChildNodes(element); // TODO: should this be changed to realChildElements.
```

Comment:
Flagging for pre-submit visibility since there's no linked issue yet

Reply:
Yeah, my instinct is that this might be a bug? Unsure

Reply:
Not a bug since text nodes etc simply won't match and will have `continue` called. Still semantically weird. Opened a PR https://github.com/ampproject/amphtml/pull/37352